evaporate.js

```diff
@@ -1364,6 +1365,10 @@
       onProgress: this.onProgress.bind(this)
     };
 
+    if(/Edge\/\d+/.test(navigator.userAgent) && fileUpload.contentType) {
```

Comment:
EvaporateJS should not do any browser sniffing. Will this addition, less the browser sniffing, work for any and all browsers?